Corona-Warn-App/src/deviceForTesters/res/layout/fragment_test_submission.xml

```diff
@@ -35,6 +35,44 @@
                 android:layout_marginTop="8dp"
                 tools:text="No OTP" />
 
+            <TextView
+                android:id="@+id/submission_time"
+                style="@style/TextAppearance.AppCompat.Caption"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="8dp"
+                tools:text="Submission Time" />
+
+            <com.google.android.material.materialswitch.MaterialSwitch
+                android:id="@+id/check_device_time_switch"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="10dp"
+                android:checked="true"
+                android:text="Check Device Time" />
+
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginBottom="10dp"
+                android:text="Note: This checkbox has effect only as long as you are in this test menu. it does not have an effect on real submission time check." />
```

Comment:
Technically, it's a switch, not a checkbox

Reply:
will leave that to Test menu changes ticket , but you understood  right 😄  ?

Reply:
And couldn't we make this switch work outside of the test menu? How would we test the flow when we keep getting `TIME_SINCE_ONBOARDING_UNVERIFIED`?

Reply:
any changes will be addressed by https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14182